app/assets/stylesheets/application.css.scss

```diff
@@ -48,6 +48,19 @@
   position: relative;
 }
 
+.admin-photo-container {
+  background-image: url('https://dancer.com/wp-content/uploads/2013/10/Cross-Training.jpg');
+  background-position: center center;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  -webkit-background-size: cover;
+  background-size: cover;
+  height: 100vh;
+  position: relative;
+  border-top: 1px;
+  border-color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.